src/test/java/org/sagebionetworks/web/unitclient/place/SynapsePlaceTest.java

```diff
@@ -105,6 +106,18 @@ public void testDiscussionVersionCase() {
 		Assert.assertNull(place.getAreaToken());
 		Assert.assertEquals(testToken, tokenizer.getToken(place));
 	}
+
+	@Test
+	public void testDockerVersionCase() {
+		String testToken = testEntityId + VERSION_DELIMITER + testVersionNumber + DOCKER_DELIMITER;

```

Comment:
I wonder if versioned docker area case is a real world use case (since they will currently live at the project level, which is not versionable).